src/Mesh/Grids.jl

```diff
@@ -228,7 +229,7 @@ function computegeometry(topology::AbstractTopology{dim}, D, ξ, ω, meshwarp,
   Canary.creategrid!(X..., topology.elemtocoord, ξ)
 
   @inbounds for j = 1:length(x)
-    (x[j], y[j], z[j]) = meshwarp(x[j], y[j], z[j])
+      (x[j], y[j], z[j]) = meshwarp(x[j], y[j], z[j])
```

Comment:
No need for changing spacing (`src/Mesh/Grids.jl` can just be reverted to master, no changes needed for this pull request)

Reply:
This may be a matter of different editors.

Reply:
My main point was that unnecessary changes should not be included in a commit/pull request to keep things cleaner. The two changes to this file have nothing to do with the PR. Only included the changes you need to in PR makes it much easier to review.

Best to avoid using `git commit -a` as you will pick up unneeded file additions like this. Use instead `git add` or even better `git add -p`.

Also, technically the [CLIMA coding convention](https://climate-machine.github.io/CLIMA/latest/CodingConventions/) stipulates 2 spaces (not the julia 4 spaces) so you may want to adjust your editor settings accordingly.

Reply:
Should I use ```git add -p ONLY_THE_FILE_TO_COMMIT``` instead of ```git commit -am 'message'``` ?